examples/sockeye_translate/README.md

```diff
@@ -0,0 +1,71 @@
+# sockeye-serving
+This example shows how to serve Sockeye models for machine translation.
+The custom handler is implemented in `sockeye_service.py`.
+Since Sockeye has many dependencies, it's convenient to use Docker.
+For simplicity, we'll use a pre-trained model and make some assumptions about how we preprocess the data.
+
+## Getting Started With Docker
+Pull the latest Docker image:
+```bash
+docker pull jwoo11/sockeye-serving
+```
+
+Download the example model archive file (MAR).
+This is a ZIP archive containing the parameter files and scripts needed to run translation for a particular language:
+* https://www.dropbox.com/s/pk7hmp7a5zjcfcj/zh.mar?dl=0
+
+Extract the MAR file to `/tmp/models`.
+ We'll use this directory as a bind mount for Docker:
+```bash
+unzip -d /tmp/models/zh zh.mar
+```
+
+Start the server:
+```bash
+docker run -itd --name mms -p 8080:8080 -p 8081:8081 -v /tmp/models/:/models jwoo11/sockeye-serving serve
```

Comment:
The default config.properties in the docker will load models from /opt/ml/model
It's better keep it consistent:

docker run -itd --name mms -p 8080:8080 -p 8081:8081 -v /tmp/models:/opt/ml/model jwoo11/sockeye-serving